cmake_common_scripts/cmake/cmake_tools.cmake

```diff
@@ -302,6 +302,28 @@ macro(target_cxx_version target)
   endif()
 endmacro()
 
+# This extracts package name and version to the following variables
+# ${prefix}_extracted_name and ${prefix}_extracted_version
+# This was taken from https://github.com/ros-industrial/abb_robot_driver, author Jon Tjerngren
```

Comment:
Should we note here that this is BSD-3 instead of Apache 2?

Reply:
@jontje I took some code from your abb repository which strips out the cmake code for extracting package name and version form the package.xml. Do you have an issues with using the Apache 2 license or do you prefer me to add BSD 3 also to this repo? @gavanderhoorn Any Thoughts?